manim/mobject/graphing/probability.py

```diff
@@ -309,7 +309,8 @@ def __init__(
         if self.bar_names is not None:
             self._add_x_axis_labels()
 
-        self.y_axis.add_numbers()
+        if self.y_axis.include_numbers:
+            self.y_axis.add_numbers()
```

Comment:
Inside the parent class `Axes`, there is already some code which builds a `NumberLine` for each axis, where numbers are added to them when the respective config is set to `True`.

Therefore, having these two lines in `BarChart` would cause numbers to be added twice.

I would suggest removing these lines and letting the adding of numbers automatically happen in the parent class.
```suggestion
```